tests/tests/test-staking/test-staking-consts.ts

```diff
@@ -59,7 +59,14 @@ describeDevMoonbeam("Staking - Consts - MaxDelegationsPerDelegator", (context) =
         context.createBlock(
           randomCandidatesChunk.map((randomCandidate) =>
             context.polkadotApi.tx.parachainStaking
-              .delegate(randomCandidate.address, MIN_GLMR_DELEGATOR, 1, maxDelegationsPerDelegator)
+              .delegateWithAutoCompound(
```

Comment:
Probably missed it changing it back, initially the chunks were failing with `Transaction Ancient` error since the txs were generated before the `createBlock` and we could only fit at most `8`, so I was instead using `delegateWithAutoCompound` since the `delegate` method is set for removal now (I didn't wanna remove it in this PR). I can change it back and generate tx within the chunk but it's pretty inconsequential since the next RT should effectively swap out all `delegate` calls with `delegateWithAutoCompound`